packages/firestore/lib/index.d.ts

```diff
@@ -499,7 +517,7 @@ export namespace FirebaseFirestoreTypes {
      *
      * @param fieldPath The path (e.g. 'foo' or 'foo.bar') to a specific field.
      */
-    get(fieldPath: string | FieldPath): any;
+    get<fieldType extends DocumentFieldType>(fieldPath: string | FieldPath): fieldType;
```

Comment:
Shouldn't this just be `get(fieldPath: string | FieldPath): DocumentFieldType;`? `fieldType` doesn't actually extend `DocumentFieldType` with any further properties

Reply:
It defines a new named type to be specifically repeated for the returning value. Is there a different way that you're supposed to do that?